ccm_web/client/src/App.tsx

```diff
@@ -52,24 +46,33 @@ function App (props: AppProps): JSX.Element {
     )
   }
 
+  const HomeBreadcrumb = (isLink: boolean): JSX.Element => {
+    const typography = (<Typography className={classes.breadcrumbs} color='textPrimary'>
+                          Canvas Course Manager
+                        </Typography>)
+    return isLink
+      ? (<Link component={RouterLink} to='/'>{typography}</Link>)
+      : (typography)
+  }
+
   return (
     <div className='App'>
       <SnackbarProvider maxSnack={3}>
         <Router>
-          <div className={classes.breadcrumbs}>
+          <div>
             <Route>
               {({ location }) => {
                 const pathnames = location.pathname.split('/').filter(x => x)
                 return (
-                  <Breadcrumbs aria-label="breadcrumb" separator={<NavigateNextIcon fontSize="small" />}>
+                  <Breadcrumbs className={classes.breadcrumbContainer} aria-label="breadcrumb" separator={<NavigateNextIcon fontSize="small" />}>
                     {HomeBreadcrumb(pathnames.length > 0)}
```

Comment:
Is it possible to use `HomeBreadcrumb` in a JSX way? Like `<HomeBreadCrumb isLink={true}>`?

Reply:
I can but it seems like it takes more code, unless there is a better way to do it.
I had to create an interface for HomeBreadcrumb properties and accept that instead of a boolean.

Maybe there is a better way than what I tried..

Is it better with longer code that allows this 

`<HomeBreadcrumb isLink={(pathnames.length > 0)} />`

instead of

`{HomeBreadcrumb(pathnames.length > 0)}`

Reply:
```
  interface BreadcrumbProps {
    isLink: boolean
  }

  const HomeBreadcrumb = (props: BreadcrumbProps): JSX.Element => {
```

Reply:
I think this is partly (or maybe wholly) a style discussion. For functions that return JSX, I would probably expect that they be components. I noticed in a couple of other places in the PR where you use similar patterns. I don't mind the extra code (interface) and find the JSX version a little more readable, but I will leave this up to your discretion. I'm okay if you prefer doing things like this. FWIW, capitalization of HomeBreadcrumb made me also think it was a component.